.github/workflows/kafka_demo_test.yml

```diff
@@ -0,0 +1,110 @@
+name: Kafka Demo Test 
+
+on:
+  workflow_call:
+    inputs:
+      sidecar-registry:
+        description: "The registry to get the image from"
+        required: true
+        default: "mcr.microsoft.com"
+        type: string
+      key-release-image:
+        description: "The image of the SKR sidecar to use"
+        required: true
+        default: "aci/skr:2.7"
+        type: string
+      consumer-image: 
+        description: "Consumer Image"
+        required: true
+        type: string
+      producer-image: 
+        description: "Producer Image"
+        required: true
+        type: string
+      kubernetes-version:
+        description: "Kubernetes version to use for AKS cluster creation"
+        required: true
+        default: "1.28.3"
+        type: string
+      skr-kafka-demo-topic:
```

Comment:
Since this is just a demo for customers, I feel like this should be hard-coded somewhere instead.

Reply:
I can have this hardcoded.

Reply:
Ah I didn't mean for it to still be an input, rather I'd like it to be just put in the code that uses it. 

I think we have a lot of things that are inputs that aren't required if we want these pipelines to be automated. I think letting the user have control of the main yml for manual runs is great, but I also don't see us caring to change the kafka topic when we want to run this pipeline manually.

Reply:
Yeah demo topic is removed.

Reply:
This comment will be resolved by removing the workflow_call section.